extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -337,6 +346,17 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
 
     /** @private {?./safeframe-host.SafeframeHostApi} */
     this.safeframeApi_ = null;
+
+    /** @private {boolean} whether safeframe forced via tag */
+    this.forceSafeframe_ = false;
+    if ('forceSafeframe' in this.element.dataset) {
+      if (!/^(1|(true))$/i.test(this.element.dataset['forceSafeframe'])) {
```

Comment:
Why do we need to check for 1/true/True, and not just 1?

Reply:
Eh I went back and forth on this as well... I figured both could be used by publishers and easy enough to support them both.